amt/site/templates/macros/cards.html.j2

```diff
@@ -40,7 +40,7 @@
     {% elif key == "email" %}
     <a class="rvo-link rvo-link--normal" href="mailto:{{value}}">{{value}}</a>
     {% elif key == "urn" %}
-    <a class="rvo-link rvo-link--normal" href="https://minbzk.github.io/instrument-registry/instruments/urn/{{value}}">{{value}}</a>
+    <a class="rvo-link rvo-link--normal" href="https://instrument-registry.apps.digilab.network/urns/?version=latest&urn={{value}}">{{value}}</a>
```

Comment:
Nice to put the version latest already in there, even though it is not fully introduced yet, AMT will not suddenly break, although latest is the default version so maybe you could still remove it here. I am indifferent